extensions/vscode/webviews/homeView/src/components/tree/TreeItem.vue

```diff
@@ -159,13 +164,24 @@ const toggleExpanded = () => {
         line-height: 22px;
         color: inherit;
         white-space: pre;
+        min-width: 0;
+        overflow: hidden;
+        text-overflow: ellipsis;
       }
 
       .tree-item-description {
+        line-height: 22px;
```

Comment:
`line-height` has returned. Now that we are using `display: flex` here the `span` is no longer being rendered the same way. Without this the height changes, and the description wasn't on the baseline.